README.md

```diff
@@ -191,15 +249,69 @@ Compile translation files:
 ./manage.py compilemessages
 ```
 
+Collect static files:
+
+```
+./manage.py collectstatic
+```
+
 You can now deactivate the environment and go back to the root session:
 
 ```
 deactivate && exit
 ```
 
+### Configure worker
+
+To execute asynchronous tasks, back as the 'root' user, create a systemd service file to run the Celery worker. In `/etc/systemd/system/accesspoc-worker.service`, with the following content:
+
+```
+[Unit]
+Description=Accesspoc Celery worker
+After=network.target
+
+[Service]
+Type=forking
+User=accesspoc
+Group=accesspoc
+EnvironmentFile=/home/accesspoc/accesspoc-env
+Environment=CELERY_BIN=/home/accesspoc/dip-access-interface/venv/bin/celery
+Environment=CELERYD_PID_FILE=/home/accesspoc/accesspoc-worker.pid
+Environment=CELERYD_LOG_FILE=/home/accesspoc/accesspoc-worker.log
+WorkingDirectory=/home/accesspoc/dip-access-interface/accesspoc
+ExecStart=/bin/sh -c '${CELERY_BIN} multi start accesspoc-worker \
+            -A accesspoc \
+            --pidfile=${CELERYD_PID_FILE} \
+            --logfile=${CELERYD_LOG_FILE} \
+            --loglevel=WARNING'
+ExecReload=/bin/sh -c '${CELERY_BIN} multi restart accesspoc-worker \
+            -A accesspoc \
+            --pidfile=${CELERYD_PID_FILE} \
+            --logfile=${CELERYD_LOG_FILE} \
+            --loglevel=WARNING'
+ExecStop=/bin/sh -c '${CELERY_BIN} multi stopwait accesspoc-worker \
+            --pidfile=${CELERYD_PID_FILE}'
```

Comment:
Is `sh` needed?

Reply:
I guess not, will check.

Reply:
It's needed when using an env. var. to declare the path, as `${...` is not recognized as an absolute path, I'll not use the env. var. for the executable.